README.md

```diff
@@ -51,21 +51,21 @@ Dependencies between packages in various layers of the system are enforced via a
 [docs/PACKAGES.md](./docs/PACKAGES.md).
 
 ## Building
-
-In order to build the Fluid Framework, ensure that you have installed [Git](https://git-scm.com/downloads) and
-[Node.js](https://nodejs.org/).
-
-Note: we recommend using nvm (for [Windows](https://github.com/coreybutler/nvm-windows) or
-[MacOS/Linux](https://github.com/nvm-sh/nvm)) to install Node.js, in case you find yourself needing to install different
-versions of Node.js side-by-side.
-
-Clone a copy of the repo and change to the repo root directory:
+Ensure [Git](https://git-scm.com/downloads) is installed, then clone a copy of the git repo and change to the repo root directory:
 
 ```shell
```

Comment:
Users need to clone the repo in order to fetch the '.nvmrc' file before running `nvm install`.

I've reordered a bit to make that clearer, as well as strengthened the recommendation to use NVM.